packetbeat/packetbeat.full.yml

```diff
@@ -21,33 +21,33 @@ packetbeat.interfaces.device: any
 # * pf_ring, which makes use of an ntop.org project. This setting provides the
 # best sniffing speed, but it requires a kernel module, and it's Linux-specific.
 # The default sniffer type is pcap.
-#packetbeat.interfaces.type: pcap
+packetbeat.interfaces.type: pcap
 
 # The maximum size of the packets to capture. The default is 65535, which is
 # large enough for almost all networks and interface types. If you sniff on a
 # physical network interface, the optimal setting is the MTU size. On virtual
 # interfaces, however, it's safer to accept the default value.
-#packetbeat.interfaces.snaplen: 65535
+packetbeat.interfaces.snaplen: 65535
 
 # The maximum size of the shared memory buffer to use between the kernel and
 # user space. A bigger buffer usually results in lower CPU usage, but consumes
 # more memory. This setting is only available for the af_packet sniffer type.
 # The default is 30 MB.
-#packetbeat.interfaces.buffer_size_mb: 30
+packetbeat.interfaces.buffer_size_mb: 30
 
 # Packetbeat automatically generates a BPF for capturing only the traffic on
 # ports where it expects to find known protocols. Use this settings to tell
 # Packetbeat to generate a BPF filter that accepts VLAN tags.
-#packetbeat.interfaces.with_vlans: true
+packetbeat.interfaces.with_vlans: true
 
 # Use this setting to override the automatically generated BPF filter.
-#packetbeat.interfaces.bpf_filter:
+packetbeat.interfaces.bpf_filter:
```

Comment:
Yes, that would be an empty string. Isn't that also the default if it is not set? https://github.com/elastic/beats/blob/master/packetbeat/beater/packetbeat.go#L196